test/helper.rb

```diff
@@ -9,11 +9,56 @@
 Test::Unit::TestCase.extend(Fluent::Test::Helpers)
 
 module TestHelperModule
+  def sample_start_log
```

Comment:
If this functions are only used by the acceptance test, they belong to that file